sdk/include/psdk/mapidefs.h

```diff
@@ -161,7 +158,7 @@ typedef struct _MAPIUID
 #define MAPI_USE_DEFAULT              0x00000040U
 #define MAPI_UNICODE                  0x80000000U /* Strings in this call are Unicode */
 
-#if defined (UNICODE) || defined (__WINESRC__)
+#ifdef UNICODE
```

Comment:
Funnily enough, Wine itself has the `__WINESRC__` guard there :joy:
https://github.com/wine-mirror/wine/blob/master/include/mapidefs.h#L150
I would probably suggest you to keep it, in this case.